rasa/utils/tensorflow/models.py

```diff
@@ -424,6 +487,28 @@ def _log_metrics_for_tensorboard(
                     if metric.name in self.metrics_to_log:
                         tf.summary.scalar(metric.name, metric.result(), step=step)
 
+    def _update_best_metrics_so_far(self, current_results: Dict[Text, Text]) -> bool:
```

Comment:
As we are returning a boolean, I would rename the method to something like `does_model_improve`